tests/events_test.go

```diff
@@ -0,0 +1,195 @@
+package tests_test
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	corev1 "k8s.io/api/core/v1"
+	k8sv1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	v1 "kubevirt.io/client-go/api/v1"
+	"kubevirt.io/client-go/kubecli"
+	"kubevirt.io/kubevirt/tests"
+	"kubevirt.io/kubevirt/tests/flags"
+)
+
+const (
+	ioerrorPV  = "ioerror-pv"
+	ioerrorPVC = "ioerror-pvc"
+	deviceName = "errdev0"
+	diskName   = "disk0"
+)
+
+var _ = Describe("[Serial][owner:@sig-compute]KubeVirtConfigmapConfiguration", func() {
```

Comment:
I  would rename the description because it doesn't depend on KubeVirtConfigmapConfiguration.